feeder/cmd/rekor_feeder/main.go

```diff
@@ -0,0 +1,195 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// rekor_feeder is an implementation of a witness feeder for the Sigstore log: Rekór.
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/google/trillian-examples/formats/log"
+	"github.com/google/trillian-examples/internal/feeder"
+	"github.com/google/trillian-examples/serverless/config"
+	"golang.org/x/mod/sumdb/note"
+
+	i_note "github.com/google/trillian-examples/internal/note"
+	wit_http "github.com/google/trillian-examples/witness/golang/client/http"
+	yaml "gopkg.in/yaml.v2"
+)
+
+var (
+	configFile = flag.String("config_file", "", "Path to feeder config file.")
+	timeout    = flag.Duration("timeout", 10*time.Second, "Maximum time to wait for witnesses to respond.")
+	interval   = flag.Duration("interval", time.Duration(0), "Interval between attempts to feed checkpoints. Default of 0 causes the tool to be a one-shot.")
+)
+
+// Config encapsulates the feeder config.
+type Config struct {
+	// Logs defines the source logs to feed from.
+	Logs []config.Log `yaml:"Logs"`
```

Comment:
For the sake of simplicity this can be made a single Log. I'm not aware of any plans for SigStore to be running multiple logs, nor anyone else adopting their scheme.

Reply:
I was going to do that originally, but they do actively encourage people to run their own sigstore/rekór instances so I ended up leaving it in.

I'm happy to remove it (knowing that we might need to re-add it) if you feel strongly about it though.